mirage/apt/apt_inputs.py

```diff
@@ -878,6 +882,9 @@ def get_filters(pointing_info):
 
             filter_list = list(set(short_pupils))
             filter_list.remove('CLEAR')
+            for wfsc_optic in  ['WLP8', 'WLM8', 'GDHS0', 'GDHS60']:
+                if wfsc_optic in filter_list:
+                    filter_list.remove(wfsc_optic)
 
             filter_list.extend(list(set(long_pupils)))
```

Comment:
I think it makes sense to do something similar to `long_pupils` and remove GRISMR and GRISMC.